warehouse/query-core/src/test/java/datawave/query/jexl/visitors/IteratorBuildingVisitorTest.java

```diff
@@ -169,10 +170,10 @@ public void NeTest() throws Exception {
         Key hit = new Key("row", "dataType" + Constants.NULL + "123.345.456");
         
         List<Map.Entry<Key,Value>> source = new ArrayList<>();
-        source.add(new AbstractMap.SimpleEntry(
-                        new Key("row", "fi" + Constants.NULL + "F1", "v0" + Constants.NULL + "dataType" + Constants.NULL + "123.345.456"), new Value()));
-        source.add(new AbstractMap.SimpleEntry(
-                        new Key("row", "fi" + Constants.NULL + "F1", "v1" + Constants.NULL + "dataType" + Constants.NULL + "123.345.456"), new Value()));
+        source.add(new AbstractMap.SimpleEntry(new Key("row", ShardFamilyConstants.FI + Constants.NULL + "F1", "v0" + Constants.NULL + "dataType"
```

Comment:
If we're going to make broad changes, I would have a preference for static imports. 

Quick plug for `FI_PREFIX` and static imports
```java
new Key("row", FI_PREFIX + "F1", "v0" + NULL + "dataType"+ NULL + "123.345.456");
```

Reply:
plug noted